plasma-roundup/draft-plasma-article.md

```diff
@@ -0,0 +1,76 @@
+
+In August 2017, Plasma creators Joseph Poon and Vitalik Buterin [proposed this framework](https://plasma.io/plasma.pdf) for scaling Ethereum transaction throughput to a "significant amount of state updates per second," potentially more than Paypal, Visa or other widely used merchant service providers.
+
+Aside from offering comparable, if not more, transactions per second Plasma stakes its security in the value of Ethereum’s decentralized mainchain rather than in a centralized merchant service.
+
+The promise of Plasma lays in its potential to help scale blockchain technology by processing a substantial amount of decentralized financial applications worldwide.
+
+In January 2018, Vitalik posted the [“Minimal Viable Plasma” (MVP) specification to the Ethereum Research Forums](https://ethresear.ch/t/minimal-viable-plasma/426). The specification was designed to offer simplicity and basic security properties to kickstart development. Teams immediately began building their own implementations.
+
+This overview is not meant to be exhaustive. Instead, by highlighting a few implementations, it aims to be indicative of the progress that Plasma has made over the past year.
+
+Although none of the Plasma models reviewed here are on mainnet or production ready, they demonstrate that this technology is not just theory. Taken together, the many implementations suggest that Plasma is moving rapidly toward realizing the scaling potential that its creators and implementers envision.     
+
+Among the early Plasma implementations was [FourthState Labs](https://github.com/FourthState/plasma-mvp-rootchain), whose design included a rootchain contract according to the Plasma MVP. This rootchain, which has been incorporated into other implementations, is pretty simple to test. To test, I used this environment:
+
+* Linux (Debian Stretch)
+* Truffle (v5.0.2)
+* Ganache CLI (v6.2.5)
+* NPM 6.7.0
+
+Follow these steps to test:
+
+0. `$ git clone https://github.com/fourthstate/plasma-mvp-rootchain`
+1. `$ cd plasma-mvp-rootchain`
+2. `$ npm install`
+3. `$ npm install -g truffle ganache-cli` // if not installed already
+4. `$ ganache-cli` // run as a background process
+5. `$ npm test`
+
+![FourthState tests](/images-for-article/Fourth-Estate/fourth-estate.png)
+
+Other notable early MVP's include [OmiseGO's research implementation](https://github.com/omisego/plasma-mvp), which has a root chain, child chain and a client to interact with the Plasma chain. Moving beyond research, [Kyokan](https://github.com/kyokan/plasma) introduced their Golang implementation [extending the original MVP specification](https://kauri.io/article/7f9e1c04f3964016806becc33003bdf3/v4/minimum-viable-plasma-the-kyokan-implementation). Kyokan uses the FourthState rootchain contract reviewed above. The architecture includes root nodes to process transactions and package them into blocks, broadcasts blocks to validator nodes, processes exits and more.
```

Comment:
@cspannos OmiseGo seems to have moved here - https://github.com/omisego/plasma-contracts

Any any chance of a bit more detail?

Reply:
@ChrisChinchilla Yes, absolutely. I took a crack at it. Let me know if you think there should be more or if you have anything specific in mind.